.gitlab-ci.yml

```diff
@@ -228,7 +228,7 @@ node-exits:
   except:
     - /^v[0-9]+\.[0-9]+.*$/        # i.e. v1.0, v2.1rc1
   script:
-    - ./ci/check_for_exit.sh
+    - ./scripts/check_for_exit.sh
```

Comment:
Why? This is a `ci` script and nothing that should be run anywhere.

Reply:
in an attempt to clean up in general, I'd like to remove both folders from root.

Reply:
Where should they go?

Reply:
moving into `.maintain`. together with other similar files.

semantic naming rather than functional - especially since more than a third are not scripts but configuration files in the first place.